lib/stop.js

```diff
@@ -0,0 +1,5 @@
+(function() {
+  'use strict';
+
+  console.log('stopping');
+})();
```

Comment:
Does this file actually do anything besides log console output? Or will it in the future?